test/Sentry.Tests/SentryClientTests.CaptureEvent_BeforeEventThrows_ErrorToEventBreadcrumb.verified.txt

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    timestamp: DateTimeOffset_1,
```

Comment:
[We've identified some problems with deserialization of the protocol](https://github.com/getsentry/sentry-dotnet/pull/616/).

Not sure it's related to this, I assume `Verify` just compares the loaded json at the end and doesn't rely on the type being able to deserilize though.

Reply:
Verify serializes the type here to avoid having to match on dates?

Reply:
Verify automatically scrubs any data that might be unstable between testruns/machines/build directory. all of these can be disabled if u want https://github.com/VerifyTests/Verify/blob/master/docs/serializer-settings.md#guids-are-scrubbed

Reply:
for guids and datetimes, they are detected and replaced with consistent values. so, for example, a given datetime will always be replaced with the same text.

Reply:
Got it. Thanks for the pointer. I think we'll disable the scrubbing for these and make sure we use consistent values (we already do that actually)

Reply:
i checked and the datetime is not consistent between runs. should it be?

Reply:
Yes, through the code base types have at least an internal overload that takes timestamp, guid etc just for testing sake.

Since we don't have round trip tests so far, it's possible it creates a UtcNow for example but then asserts against that. Going forward, for the sake of snapshot tests that approach can't be used anymore and a fixed date should be used (that includes some level of precision to cover the scenario of millseconds when relevant)